src/init/inst-functions.sh

```diff
@@ -1117,6 +1117,18 @@ TransferShared()
     find ${INSTALLDIR}/etc/shared -maxdepth 1 -type f -not -name ar.conf -not -name files.yml -exec mv -f {} ${INSTALLDIR}/etc/shared/default \;
 }
 
+checkDownloadContent()
+{
+    VD_FILENAME='vd_1.0.0_vd_4.8.0.tar.xz'
+
+    if [ "${DOWNLOAD_CONTENT}" = "yes" ]; then
+        echo "Download ${VD_FILENAME} file"
+        wget -O ${VD_FILENAME} https://.../${VD_FILENAME}
+
+        ${INSTALL} -m 0640 -o ${WAZUH_USER} -g ${WAZUH_GROUP} ${VD_FILENAME} ${INSTALLDIR}/
```

Comment:
The installation by sources doesn't decompress the file.
Also, the root install directory might not be the correct place for this file.

Reply:
Done

Reply:
This is the src folder